packages/node_modules/@ciscospark/plugin-people/src/people-batcher.js

```diff
@@ -9,6 +9,9 @@ import {Batcher} from '@ciscospark/spark-core';
  */
 const PersonUUIDRequestBatcher = Batcher.extend({
   namespace: `People`,
+  // optional flag that requires Hydra to send every type field,
```

Comment:
Does `showAllTypes` add the `type` field to the response, or does it cause the response to include more than just `type == person` people?

Reply:
The "showAllTypes" flag will include the "type" field in cases where it otherwise would have been omitted. For example, in the case of a "webhook_integration" the type field is omitted entirely without the inclusion of the additional query parameter "showAllTypes=true".

The type field for "person" will always be included without any special query parameters necessary.